instruqt-tracks/terraform-intro-aws/terraform-graph/solve-workstation

```diff
@@ -6,7 +6,6 @@ set -o history
 
 cd /root/hashicat-aws
 
-# This needs to be fixed so it runs cleanly.
```

Comment:
With the fixes to how we install `BlastRadius` this comment is no longer applicable.